File: Task 4/Java/app/src/main/java/dev/ogabek/java/manager/PrefsManager.java
package dev.ogabek.java.manager;

import android.content.Context;
import android.content.SharedPreferences;

public class PrefsManager {

    public SharedPreferences sharedPreferences;

    public PrefsManager(Context context) {
       sharedPreferences  = context.getSharedPreferences("MyPref", Context.MODE_PRIVATE);
    }

    public void saveData(String key, String value) {
        SharedPreferences.Editor prefsEditor = sharedPreferences.edit();
        prefsEditor.putString(key, value);
        prefsEditor.apply();
    }

    public String getData(String key) {
        return sharedPreferences.getString(key, "en");
    }

}

File: Task 4/Kotlin/app/src/main/java/dev/ogabek/kotlin/manager/PrefsManager.kt
package dev.ogabek.kotlin.manager

import android.content.Context
import android.content.SharedPreferences

class PrefsManager(context: Context) {
    var sharedPreferences: SharedPreferences = context.getSharedPreferences("MyPref", Context.MODE_PRIVATE)
    fun saveData(key: String?, value: String?) {
        val prefsEditor = sharedPreferences.edit()
        prefsEditor.putString(key, value)
        prefsEditor.apply()
    }

    fun getData(key: String?): String? {
        return sharedPreferences.getString(key, "en")
    }

}